content/english/python-basics/lists/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Lists"
+draft: false
+weight: 10
+---
+
+Lists are the ordered collection of items and can be written as comma separated values inside square bracket `[]`. You can access the element  of a list by using indexes inside square bracket. The index starts from `0` and goes upto the `(number of elements in the list - 1)` and the first element is at index `0`. So, a lis having 6 elements will have indices from 0 to 5. 
+
+```python
+#Creating a list of 5 fruits
+fruits = ['orange', 'apple', 'banana', 'apple', 'mango']
+
+#Print element at index 1
+print(fruits[2])     ##it prints banana
+
+
+```
+
+![](listDiagram.png)
+
+{{% notice note %}}
+
+Lists can have duplicate elements.
+
+{{% /notice %}}
+
+```python
+#lists can also have different datatypes
+my_list = ['candy', 10, 2.5]
+
+```
+
+### Length of the list
+
+You can get the length of the list by using `len(list)` function and the list name is passed as the parameter.
+
+```python
+
+length = len(fruits)  ##Now length = 5
+
+
+```
+
+### Append an element in the list
+
+Appending an element can be done by the `list.append(x)` function where `x` is the element that you want to append. It adds the element at the end of the list.
+
+```python
+
+fruits.append('lemon')
```

Comment:
I think it'd be nice to show what the `fruits` array has before doing `append` just so that the student doesn't have to keep scrolling up to remember what the `fruits` array had:

```suggestion
#Before the list is ['orange', 'apple', 'banana', 'apple', 'mango']
fruits.append('lemon')
```